commitizen/commands/commit.py

```diff
@@ -28,7 +28,10 @@ def __init__(self, config: BaseConfig, arguments: dict):
         self.config: BaseConfig = config
         self.cz = factory.commiter_factory(self.config)
         self.arguments = arguments
-        self.temp_file: str = os.path.join(tempfile.gettempdir(), "cz.commit.backup")
+        name = "cz.commit.backup"
+        if "USER" in os.environ:
+            name = name + "." + os.environ['USER']
```

Comment:
I'd prefer something like this:
```
user = os.environ.get('USER', "")
name = f"cz.commit{user}.backup
```

Reply:
Thank you for the apply. we have 100+ users project using this package which have this issue, but I am new commiter for this python package. I will  rebuild this issue only after work.

Reply:
No rush, take your time